src/m365/planner/commands.ts

```diff
@@ -3,6 +3,7 @@ const prefix: string = 'planner';
 export default {
   BUCKET_ADD: `${prefix} bucket add`,
   BUCKET_LIST: `${prefix} bucket list`,
+  BUCKET_GET: `${prefix} bucket get`,
```

Comment:
Should be places above `bucket list` to maintain alphabetical order.

Reply:
Thanks, I did not think about order.